app/javascript/actions/eventActions.ts

```diff
@@ -1,12 +1,13 @@
 import Event from "../models/Event";
-import { IEvent, IPeriod } from "../models/Event";
+import { IEvent, IPeriod, ITag } from "../models/Event";
 import { IPerson } from "../models/Person";
 
 export const ADD_EVENTS = "ADD_EVENTS";
 export const ADD_EVENTS_FOR_LANGUAGE = "ADD_EVENTS_FOR_LANGUAGE";
 export const ADD_EVENTS_FOR_PERSON = "ADD_EVENTS_FOR_PERSON";
 export const SET_EVENT = "SET_EVENT";
 export const DELETE_EVENT = "DELETE_EVENT";
+export const ADD_TAG = "ADD_TAG";
```

Comment:
This doesn't look like an action we should need, and I haven't seen if it's actually used...

Ideally, the added tag is kept in component state until we click Save, then we dispatch a LoadAction with the server response.